### PR TITLE
CCPi patch script

### PR DESCRIPTION
**Pull request by paskino**
Adds a patch script for the CCPi-Framework installation.

### CODE REVIEW

File: patches/cil-patch.py [hunk 1]
```diff
@@ -0,0 +1,8 @@
+from __future__ import print_function, division
```
Comment (KrisThielemans) [inline patches/cil-patch.py]: I guess you don't need this line

Comment (paskino) [reply patches/cil-patch.py]: ah! you know, it's safe anyway.

**Comment by KrisThielemans** [comment]
restarted the last Travis job. network problems

**Comment by paskino** [comment]
of course the patch fails on the VM with git paranoia: the patch changes the file in the repo under git control

**Comment by paskino** [comment]
Works now.

**Pull request merged**